src/main/scala/reporter/RemoraDatadogReporter.scala

```diff
@@ -0,0 +1,40 @@
+package reporter
+
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
+import config.DataDog
+import org.coursera.metrics.datadog.DatadogReporter
+import org.coursera.metrics.datadog.transport.UdpTransport
+
+/**
+  *
+  * @param metricRegistry The metrics registry containing the metrics
```

Comment:
I was actually asking to remove this entire comment block. We currently don't release scala docs, but if we ever do enterprise this we can put them in.

After removing this, lets merge

Reply:
Sorry for the misunderstanding, it is removed